cfg/windows.cfg

```diff
@@ -3628,6 +3638,102 @@ HFONT CreateFont(
       <not-uninit/>
     </arg>
   </function>
+  <!--HMODULE WINAPI LoadLibrary(
+  _In_ LPCTSTR lpFileName);-->
+  <function name="LoadLibrary,LoadLibraryA,LoadLibraryW">
+    <noreturn>false</noreturn>
+    <returnValue type="HMODULE"/>
```

Comment:
a use-retVal element can also be added

Reply:
I thought about it, but since the LoadLibrary function is marked as an <alloc> function cppcheck already outputs a warning if the handle is immediately lost. I was not sure if it is useful to add <use-retval/> because then there are nearly duplicate messages. But maybe it is ok to add it, just to be sure?

Reply:
I've added use-retval to the LoadLibrary* functions now.
If the return value for a LoadLibrary call is not used one will get these two messages:
(warning) Return value of function LoadLibrary() is not used.
(error) Return value of allocation function 'LoadLibrary' is not stored.

Reply:
ok thanks for the carification.. I guess I revert that then for now. I think the checkers should make sure that both messages are not written.